ansible/kubernetes.yml

```diff
@@ -9,12 +9,26 @@
       listen:
         - systemd_read_configs
   post_tasks:
+    - name: ansible_debug
+        This task is executed by vagrant provision --provision-with ansible-debug
```

Comment:
Are you sure this format works?

Btw, I'd rephrase this to avoid mentioning vagrant, because it's already in the readme, and to avoid passive forms. Apart from that, this is a great help